server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -6,6 +6,7 @@
 require_once "$root/vendor/autoload.php";
 require_once "$root/server/utilities/emailUtilities.class.php";
 require_once "$root/server/utilities/appointmentTypeUtilities.class.php";
+require_once "$root/server/accessors/appointmentAccessor.class.php";
```

Comment:
Is this all it takes to include the accessor? This is all I did and it works but I have no clue how all the pieces of this site are glued together so I wanted to make sure.

Reply:
Yep that's all it takes--this `require_once` is PHP's mechanism for importing other files / their functions